src/hb-subset-plan.cc

```diff
@@ -91,94 +91,102 @@ _remap_indexes (const hb_set_t *indexes,
 typedef void (*layout_collect_func_t) (hb_face_t *face, hb_tag_t table_tag, const hb_tag_t *scripts, const hb_tag_t *languages, const hb_tag_t *features, hb_set_t *lookup_indexes /* OUT */);
 
 
-template <typename T>
-static void _collect_layout_indices (hb_face_t		  *face,
-                                     const T&              table,
-                                     const hb_set_t	  *layout_features_to_retain,
-                                     layout_collect_func_t layout_collect_func,
-                                     hb_set_t		  *indices /* OUT */)
+/**
```

Comment:
Comments starting with `/**` are interpreted by gtk-doc, which I think not the intention here.